src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
+
+spider.style.top = '50%';
+spider.style.left = '50%';
+spider.style.transform = 'translate(-50%, -50%)';
```

Comment:
use some of these properties to get values in pixels for `top` and `left`
![image](https://user-images.githubusercontent.com/98764641/233935754-079bd571-38c8-427f-9c39-2487604d2c11.png)